targets/arduino-zero-programming.json

```diff
@@ -0,0 +1,9 @@
+{
+    "inherits": ["atsamd21g18a"],
+    "build-tags": ["sam", "atsamd21g18a", "arduino_zero"],
+
+    "flash-method":"openocd",
+    "openocd-transport": "swd",
+    "openocd-target": "at91samdXX",
+    "openocd-interface":"cmsis-dap"
+}
```

Comment:
I don't think this file is necessary.
I think it is possible to flash without arduino-zero-programming.json

```
tinygo flash -target=arduino-zero-native --size short --programmer cmsis-dap examples/blinky2
```

And if you delete this file, arduino-zero-native.json should be renamed to arduino-zero.json.

Reply:
Hi @sago35 I was initially trying to add support for programming via either port on the Zero.  I've only managed partial functionality for the programming port anyway so I'll try to get your example working!